### DISCUSSION TITLE
Automation issues

### DISCUSSION THREAD

**Discussion by Tctto** [RavenSystem/esp-homekit-devices#1768] [Q&A]
Anyone having any issues with Homekit automations?

All my automations have stopped working but only to flashed Sonoff devices. Genuine Homekit devices still seem to work. Have Apple put a stop to these?

**Comment by RavenSystem** [comment]
Check your setup. All automations are working fine. Apple allows this since beginning.

**Reply by Tctto** [reply]
Ok thanks, it was all working until latest IOS update a few days ago.

**Reply by Tctto** [reply]
Seems to be an issue with non certified accessories. 
The setup I have is Homepod mini on version 15.3 and Home app on Iphone on IOS 15.3.1.
These was working fine until a few days ago, which is when I did a software update.

**Comment by Tctto** [comment]
After spending a bit of time on google it seems that this is a fairly common problem, especially with IOS15. After talking to Apple support, who had me restore my phone and reset Homepod mini network settings, I was still having issues.  

I noticed when not on Wifi I couldn't control any device that didn't have its own bridge (ie all Sonoff flashed devices). So yesterday I reset Homepod to factory settings (press and hold top down for 3 beeps) and re added it to home app. At the time this didn't seem to work, but today without changing anything else they are all back up and running again!

So basically no idea what was going on or how it was fixed. Router issue? Homepod issue? no idea.
